src/backoffice/settings/languages/workspace/language/views/edit/edit-language-workspace-view.element.ts

```diff
@@ -3,10 +3,10 @@ import { UUITextStyles } from '@umbraco-ui/uui-css';
 import { css, html, nothing } from 'lit';
 import { customElement, state } from 'lit/decorators.js';
 import { ifDefined } from 'lit/directives/if-defined.js';
-import { UmbChangeEvent } from '@umbraco-cms/events';
 import { UmbLanguageWorkspaceContext } from '../../language-workspace.context';
 import { UmbLitElement } from '@umbraco-cms/element';
 import { LanguageModel } from '@umbraco-cms/backend-api';
+import { UmbChangeEvent } from '@umbraco-cms/events';
 import UmbInputCultureSelectElement from 'src/backoffice/shared/components/input-culture-select/input-culture-select.element';
 import UmbInputLanguagePickerElement from 'src/backoffice/shared/components/input-language-picker/input-language-picker.element';
```

Comment:
Use relative imports here

Reply:
I think this comes from main or Mads' PR.
I have corrected it in this branch.